config_management/group_vars/all

```diff
@@ -2,7 +2,7 @@
 go_version: 1.7.4
 terraform_version: 0.8.5
 docker_version: 1.11.2
-kubernetes_version: 1.5.2
+kubernetes_version: 1.6.0-beta.2
```

Comment:
Do you think setting an "unstable" (`beta`/`alpha`) version as a default could make our own CIT more unstable?
Not knowing the answer to this question myself, I'd rather stick to a Kubernetes release, as it would be closer to what end-users set up.